src/sinol_make/commands/run/structs.py

```diff
@@ -17,3 +17,13 @@ class ValidationResult:
 	changes: List[ResultChange]
 	expected_scores: dict
 	new_expected_scores: dict
+
+@dataclass
+class ExecutionResult:
+	# Result status of execution. Can be one of:
+	# "OK", "WA", "TL", "ML", "RE", "CE"
+	Status: str
+	# Time in milliseconds
+	Time: float
+	# Memory in Kbytes
```

Comment:
```suggestion
	# Memory in KB
```

Reply:
I don't think it should be, but there is 
`math.ceil(memory_limit)` 
and
`self.memory_limit = args.ml * 1024.0 if args.ml is not None else config_memory_limit`
for some reason

Reply:
Maybe the author also wasn't sure whether it's a float or an int. The memory is getting retrieved from oiejq, right? Maybe looking into sio2jail might find the answer.

Reply:
I read a bit and I think that memory is always an int 739360d47a75e70a998138588eb032b1b63dd1f8